src/core/evaluator.js

```diff
@@ -723,29 +723,70 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
           return errorFont();
         }
       }
+      let fallbackFont = false;
+      let fallbackFontProperties = false;
       if (!fontRef) {
-        warn('fontRef not available');
-        return errorFont();
+        // warn('fontRef not available');
+        // return errorFont();
+
+        // fallback to some default font
+        let widthsWithNulls = [null, null, null, null, null, null, null,
+        null, null, null, null, null, null, null, null, null, null, null,
+        null, null, null, null, null, null, null, null, null, null, null,
+        null, null, null, 278, 278, 355, 556, 556, 889, 667, 191, 333,
+        333, 389, 584, 278, 333, 278, 278, 556, 556, 556, 556, 556, 556,
+        556, 556, 556, 556, 278, 278, 584, 584, 584, 556, 1015, 667, 667,
+        722, 722, 667, 611, 778, 722, 278, 500, 667, 556, 833, 722, 778,
+        667, 778, 722, 667, 611, 722, 667, 944, 667, 667, 611, 278, 278,
+        278, 469, 556, 333, 556, 556, 500, 556, 556, 278, 556, 556, 222,
+        222, 500, 222, 833, 556, 556, 556, 556, 333, 500, 278, 556, 500,
+        722, 500, 500, 500, 334, 260, 334, 584];
+        let widthsWithoutNulls = [];
+        for (let i = 0; i < widthsWithNulls.length; i++) {
+          if (widthsWithNulls[i] !== null) {
+            widthsWithoutNulls[i] = widthsWithNulls[i];
+          }
+        }
+        fallbackFontProperties = {
```

Comment:
Please note that a plain `Object` isn't an appropriate data-structure here, since it for one seem to cause *a lot* of unwanted special-casing throughout the font parsing code below.

Instead you need to create a *valid* font dictionary here, with dummy entries as needed, to ensure that the fallback font dictionary can be parsed as-is by the relevant code below.